napari/components/viewer_model.py

```diff
@@ -556,10 +557,15 @@ def add_points(
             Symbol to be used for the point markers. Must be one of the
             following: arrow, clobber, cross, diamond, disc, hbar, ring,
             square, star, tailed_arrow, triangle_down, triangle_up, vbar, x.
-        size : float, array
+        size : float, tuple, list, array (N,)
             Size of the point marker. If given as a scalar, all points are made
-            the same size. If given as an array, size must be the same
-            broadcastable to the same shape as the data.
+            the same size. If given as a tuple, list, or 1-D array size must be
+            the same length as the number of points.
+        anisotropy : tuple, list, array (D,), optional
+            List of anisotropy factors, must be one per dimension. These act as
+            pre-multipliers on the size of each point. Note that the anisotropy
+            is only displayed for dimensions out of plane for the view. If not
```

Comment:
displayed -> used? The anisotropy is not really "displayed".

A clarification here about *how* it's used would be handy. Higher anisotropy means that the point will "leak" into more planes? Or fewer?